### DISCUSSION TITLE
Ignore Junction, Stay on Road

### DISCUSSION THREAD

**Discussion by MotoUKRider** [devemux86/cruiser#211] [Q&A] [answered]
I took a long drive in the car today using Cruiser for both "to" and "from" my destination - a simple A to B route (using BRouter) on 3 motorways.  What I didn't see from my casual glances was any kind of "Stay on Motorway" as I approach a junction.  The arrow for the one I needed (eg: in about 30-miles) remained visible, but a little confusing possibly.

For info: I don't use voice prompts - this is purely a display thing, so there *may* possibly be a related voice-alert I'm not familiar with.

Is there any way Cruiser can "see" the junction during navigation and offer a "Keep Left/Right" or "Stay on Motorway" type alert, or even revert the next-turn to forward arrow on approach to the junction?

I know that GoogleMaps and to some extent, CoPilot can display a mockup of the exit, with 2 lanes coming off and 2 or 3 lanes continuing - with arrows in the lane/s you should be in.  I think I've mentioned this feature before, and an example can be found [by scrolling to How Does Lane Assist work](https://support.copilotgps.com/en/support/solutions/articles/19000046170-how-do-i-setup-copilot-for-driving-) on the CoPilot help page.

**Comment by devemux86** [comment, marked as answer]
@MotoUKRider thanks for the idea!

If you mean lane guidance, indeed we have a discussion here:
- https://github.com/devemux86/cruiser/discussions/110

Note that the turn instructions are provided by the routers and Cruiser uses them in navigation.
(you can see all of them in "Turn instructions" screen)

If you mean that Cruiser will analyze a route and add extra turn instructions,
it is possible but may be more complicated.

**Reply by MotoUKRider** [reply]
Thanks :)
